packages/core/package.json

```diff
@@ -12,7 +12,7 @@
     "clean": "lb-clean loopback-core*.tgz dist package api-docs",
     "pretest": "npm run build",
     "integration": "lb-mocha \"dist/__tests__/integration/**/*.js\"",
-    "test": "lb-mocha \"dist/__tests__/unit/**/*.js\" \"dist/__tests__/integration/**/*.js\" \"dist/__tests__/acceptance/**/*.js\"",
+    "test": "lb-mocha \"dist/__tests__/**/*.js\"",
```

Comment:
This is unrelated to this pull request.

I don't have a strong opinion on how to specify test files. However, I would like us to be consistent in all packages, including our examples and CLI project templates.

Please revert this change here and open a new pull request to address the problem in a wider context.

Reply:
See https://github.com/strongloop/loopback-next/pull/2597